coverage-tests.js

```diff
@@ -2235,4 +2235,15 @@ test('should send agentRunId', {
     assert.equal(info1.startInfo.agentRunId, info2.startInfo.agentRunId)
   },
 })
+
+test('appium iOS nav bar check region', {
+  env: {device: 'iPhone XS', app: 'https://applitools.jfrog.io/artifactory/Examples/awesomeswift.zip'},
+  config: {baselineName: 'Appium_iOS_CheckRegion'},
```

Comment:
`baselineName` shouldn't be needed. It's constructed from the test name dynamically.